src/components/Header/SearchBar/SearchBar.component.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router';
+import { SearchInput } from './SearchBar.styles';
+
+const SearchBar = ({ setSearchText }) => {
+  const [inputValue, setInputValue] = useState('');
+
+  const history = useHistory();
+
+  const onChangeInput = (e) => {
+    setInputValue(e.target.value);
+  };
+
+  const handleKeyPress = (e) => {
+    if (e.key === 'Enter') {
+      setSearchText(e.target.value);
+      history.push('/');
+    }
+  };
+
+  return (
+    <SearchInput
+      value={inputValue}
+      type="text"
+      placeholder="Wizeline"
+      onChange={(e) => onChangeInput(e)}
+      onKeyPress={(e) => handleKeyPress(e)}
```

Comment:
No había visto esto en las sesiones, pero este tipo de sintaxis no es necesaria a menos que quieras pasar argumentos extras al evento, 
```suggestion
      onChange={onChangeInput}
      onKeyPress={handleKeyPress}
```
Si lo dejas así por defecto el evento se pasa a los handlers.